README.md

```diff
@@ -21,7 +21,7 @@ For more technical details, see [our wiki](http://wiki.languagetool.org).
 #### Scripted installation and building
 To install or build using a script, simply type:
 ```
-curl -L https://git.io/vFHpu | sudo bash <options>
+curl -L https://bit.ly/2kfvAha | sudo bash <options>
```

Comment:
The static link that existed before was an old version, but it is a potential security problem to have a script that requires with administrative permissions to retrieve code dynamically. That way it is impossible to review this code.

If the script is included, why should a web version be retrieved?

Reply:
So do you want me to revert back to git.io to create static links? We could also just place it on the langaugetool website and have this script fetch it from there.

Reply:
I do see how using `curl | sudo bash` can be worrying for some paranoid folks, so maybe we could sign it with our pgp keys and/or put the checksum up on github so those people can check the script themselves. Something like pipethis: https://github.com/ellotheth/pipethis
could also be used.